### PR TITLE
feat: use execute_from_outside

### PR DESCRIPTION
**Pull request by enitrat**
## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Resolves: #926 #908

## What is the new behavior?

- Defaults to using `execute_from_oustide` when executing transactions from EOA
- Requires a relayer to invoke the `execute_from_outside` function, as the `__execute__` and `__validate__` entrypoints have been disabled
- 

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

- - -
This change is [Reviewable](https://reviewable.io/reviews/kkrt-labs/kakarot-ssj/927)

### CODE REVIEW

File: crates/contracts/src/account_contract.cairo [hunk 1]
```diff
@@ -163,95 +161,17 @@ pub mod AccountContract {
...
-            } else {
-                (false, KAKAROT_VALIDATION_FAILED.span(), 0)
-            };
-            let return_data = serialize_bytes(return_data).span();
-
-            self.emit(TransactionExecuted { response: return_data, success: success, gas_used });
-
-            array![return_data]
+            panic!("EOA: __execute__ not supported");
+            array![]
```
Comment (obatirou) [inline crates/contracts/src/account_contract.cairo:174]: Array is necessary but not felt above ?

File: crates/contracts/src/account_contract.cairo [hunk 2]
```diff
@@ -312,58 +233,48 @@ pub mod AccountContract {
...
-            assert!(
-                *call.selector == selector!("eth_send_transaction"),
-                "selector must be eth_send_transaction"
-            );
+            // Kakarot-Specific Validation
+            assert(outside_execution.calls.len() == 1, 'KKRT: Multicall not supported');
+            assert(tx_info.version.into() >= 1_u256, 'KKRT: Deprecated tx version: 0');
+
+            // EOA Validation
+            assert(self.Account_bytecode_len.read().is_zero(), 'EOA: cannot have code');
```
Comment (obatirou) [inline crates/contracts/src/account_contract.cairo:241]: Is there tests for those assert ?

**Review by obatirou — DISMISSED** [review]

File: crates/contracts/src/account_contract.cairo (hunk 1 above)
Comment (enitrat) [reply crates/contracts/src/account_contract.cairo]: when returning a panic, things compile well

File: crates/contracts/src/account_contract.cairo (hunk 2 above)
Comment (enitrat) [reply crates/contracts/src/account_contract.cairo:240]: added one

**Review by obatirou — DISMISSED** [review]

**Pull request merged**